### PR TITLE
Proposed Meeting Minutes for Team Leads 2017-01-30

### PR DESCRIPTION
**Pull request by ophilli**
Instead of emailing this to you guys I'm proposing that we host it on Github - this improves transparency, and makes sure everyone has access to a consistent record.

### CODE REVIEW

**Review by gacosta1 — APPROVED** [review]

**Comment by ophilli** [comment]
@mbgroom I am waiting on your approval before merging these minutes.

**Pull request merged**
